app/assets/stylesheets/components/lesson/lesson_header.scss

```diff
@@ -6,4 +6,8 @@
     font-weight: bold;
     text-transform: uppercase;
   }
+
+  h1 {
```

Comment:
Putting the margin under the `&__title` css declaration in the banner style sheet here https://github.com/TheOdinProject/theodinproject/blob/redesign/app/assets/stylesheets/components/courses/banner.scss would be better. The h1 here is a part of the banner so keeping all styles related to in the same stylesheet will make things simpler.